packages/backend/src/api/controllers/activity/ActivityController.test.ts

```diff
@@ -23,7 +23,7 @@ describe(ActivityController.name, () => {
   describe(ActivityController.prototype.getActivity.name, () => {
     it('throws if ethereum not present in db', async () => {
       const includedIds: ProjectId[] = [PROJECT_A, ProjectId.ETHEREUM]
-      const counters: TransactionCounter[] = [
+      const counters: SequenceProcessor[] = [
```

Comment:
Just to be consistent here, also in other tests in this file.
```suggestion
      const processors: SequenceProcessor[] = [
```